Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -0,0 +1,1232 @@
+        readable_output=readable_output,
+        raw_response=json_response
+    )
+
+
+def rule_create_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Create and assign rules to a particular ruleset.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: Arguments obtained from demisto.args()
+
+    Returns:
+        CommandResults: CommandResults object
+    """
+    ruleset_href = args.get("ruleset_href")
+    providers = args.get("providers")
+    consumers = args.get("consumers")
+    ingress_services = args.get("ingress_services")
+    resolve_providers_as = args.get("resolve_providers_as", "workloads")
+    resolve_consumers_as = args.get("resolve_consumers_as", "workloads")
+
+    validate_required_parameters(ruleset_href=ruleset_href, providers=providers, consumers=consumers)
+    providers = argToList(providers)
+    consumers = argToList(consumers)
+    ingress_services = argToList(ingress_services)
+    resolve_providers_as = argToList(resolve_providers_as)
+    resolve_consumers_as = argToList(resolve_consumers_as)
+
+    # Building params to check whether rule is present in particular ruleset or not
+    params = {"ingress_services": sorted(ingress_services), "providers": sorted(providers),
+              "consumers": sorted(consumers),
+              "resolve_providers_as": sorted(resolve_providers_as), "resolve_consumers_as": sorted(resolve_consumers_as)
+              }
+
+    ruleset = client.rule_sets.get_by_reference(ruleset_href)
+    ruleset_json = ruleset.to_json()
+    for rules in ruleset_json.get("rules", []):
+        existing_rule = {"ingress_services": sorted([href.get('href') for href in rules.get("ingress_services", {})]),
+                         "providers": sorted(extract_values_from_dictionary(rules.get("providers"))),
+                         "consumers": sorted(extract_values_from_dictionary(rules.get("consumers"))),
+                         "resolve_providers_as": sorted(rules.get("resolve_labels_as").get("providers")),
+                         "resolve_consumers_as": sorted(rules.get("resolve_labels_as").get("consumers"))
+                         }
+        if params == existing_rule:
+            demisto.info("Found existing Rule bounded to the Ruleset: {}.".format(ruleset_href))
+            rule_href = rules.get("href")
+            response = client.rules.get_by_reference(rule_href)
+            break
+    else:
+        rule = Rule.build(
+            ingress_services=ingress_services,
+            consumers=consumers, providers=providers,
+            resolve_consumers_as=resolve_consumers_as,
+            resolve_providers_as=resolve_providers_as
+        )
+        response = client.rules.create(rule, parent=ruleset_href)
+
+    response = response.to_json()
+    readable_output = prepare_rule_create_output(response)  # type: ignore
+
+    return CommandResults(
+        outputs_prefix="Illumio.Rule",
```

Comment:
```suggestion
        outputs_prefix="Illumio.RuleSet",
```
not sure about this one

Reply:
> not sure about this one

Rule and ruleset are different so won't be possible to update this change